client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -20,15 +20,14 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.util.*;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```

Comment:
useless changes

Reply:
it's optimized automaticly by code format

Reply:
the optimized is determined by your idea import config, you should check it and keep align with the codebase.